src/polling-message-stream.ts

```diff
@@ -62,28 +64,18 @@ export class PollingMessageStream extends Readable {
     this._reading = false;
   }
   /**
-   * Destroys the stream and any underlying streams.
+   * Cancels any active requests once the stream is destroyed.
    *
-   * @param {error?} err An error to emit, if any.
    * @private
    */
-  destroy(err?: Error): void {
```

Comment:
With v6 reaching EOL in a few days, I decided to just remove this and wait to merge until we've dropped support for it.